robottelo/ui/rhai.py

```diff
@@ -0,0 +1,11 @@
+# -*- encoding: utf-8 -*-
```

Comment:
YAGNI. Please drop the module. You can keep this code in a separate branch in your repository.